src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -266,6 +266,10 @@ project-with-s3:
                 public: true
             widgets-encrypted-{instance}:
                 encryption: arn:aws:kms:us-east-1:1234:key/12345678-1234-1234-1234-123456789012
+            # not recommended, however 'instance' here may be derived from an alt-config name
+            # that may in turn contain MixedCase alphanumeric
+            widgets-MixedCase-{instance}:
```

Comment:
can we consider just refusing this and tell the user to change their mind? I understand this can be automatically generated from an alt config name, but shouldn't that also stick to the simplest possible combination of characters as it gets into hostnames, DNS, etc?

Reply:
I'll enforce all lowercase and change the name of the alt config

Reply:
Thank you, it's a great simplification when seeing this from the outside